libraries/AP_OSD/AP_OSD_Screen.cpp

```diff
@@ -1327,7 +1343,25 @@ void AP_OSD_Screen::draw_restvolt(uint8_t x, uint8_t y)
     backend->write(x,y, v < osd->warn_restvolt, "%c%2.1f%c", SYMBOL(SYM_BATT_FULL) + p, (double)v, SYMBOL(SYM_VOLT));
 }
 
-
+void AP_OSD_Screen::draw_avgcellrestvolt(uint8_t x, uint8_t y)
+{
```

Comment:
maybe make a draw_avgcellrestvolt_common() and pass a boolean for which to display